day_7/caesar.rb

```diff
@@ -0,0 +1,34 @@
+#
+# def to_code(string)
+# encode = 26
+# code = encode.chars.map(&:ord)
+# code = encode.chars.map {|x| x.ord}
+# moving = encode.map {|x| x + 3}
+# moving.map {|x| x.chr }.join
+# end
+#
+# puts to_code("ouch")
+
+def caesar_cipher(k)
+  alphabet = Array('a'..'z')
+  encrypter = Hash[alphabet.zip(alphabet.rotate(3))]
+  k.split('').map { |x| encrypter.fetch(x, " ") }
+end
+
+
+puts "Feed me your Password!(Or anything.)"
+password = gets.chomp
+
+puts "YUMMY!"
+p caesar_cipher("#{password}").join
+
+
+def decoder(k)
+  alphabet = Array('a'..'z')
+  decrypter = Hash[alphabet.zip(alphabet.rotate(-3))]
```

Comment:
Great work thinking through the next step of decrypting. How might you refactor encrypt/decrypt methods that use `rotate(-3)` to not always tied to shifting 3 places?